keps/sig-node/127-user-namespaces/README.md

```diff
@@ -595,6 +641,20 @@ use container runtime versions that have the needed changes.
 
 ### Upgrade / Downgrade Strategy
 
+Existing pods will still work as intended, as the new field is missing there.
+
+Upgrade will not change any current behaviors.
+
+When the new functionality wasn't yet used, downgrade will not be affected.
+
+On downgrade, when the functionality was used, the pods created with
+user namespaces that are running will continue to run with user
+namespaces. Pods will need to be re-created to stop using the user
+namespace.
+
+Versions of Kubernetes that doesn't have this feature implemented will
+ignore the new field `pod.spec.hostUsers`.
+
```

Comment:
I already commented on that in https://github.com/kubernetes/enhancements/pull/4439/#discussion_r1475820339
but it got lost.

Can you please update verskiew skew strategy to cover also kube-apiserver (that may support the new feature whereas kubelet don't have it or have FG disabled)?

Reply:
Added now, sorry! It will be in the next push